node.gyp

```diff
@@ -578,6 +578,19 @@
             'src/tls_wrap.h'
           ],
         }],
+        [ 'node_use_large_pages=="true"', {
+          'defines': [ 'NODE_ENABLE_LARGE_CODE_PAGES=1' ],
+          # The current implementation of Large Pages is under Linux.
+          # Other implementations are possible but not currently supported.
+          #
+          'conditions': [
+            [ 'OS=="linux"', {
+              'sources': [
+                'src/node_large_page.cc'
```

Comment:
could you add 'src/large_pages/node_large_page.h' as well

Reply:
@refack Addressed in the latest updated PR.